.github/CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
 ### Documentation
 
 ### Bug fixes
+ * Fix for errors caused by new lightning inheirtance implementation(DefaultQubit->QubitDevice).
+ [(#16)](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/16)
```

Comment:
I don't think we should have this reported in the changelog.
There is no fix regarding the previous release.

Reply:
I mean, reported as a bugfix. We can have it reported as one more step in the change of device inheritance.